### PR TITLE
Add phpunit and update deps.

### PR DESCRIPTION
**Pull request by simensen**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
         "silex/silex": "~1.0@dev",
         "stack/builder": "~1.0@dev",
         "stack/session": "~1.0@dev",
-        "stack/callable-http-kernel": "~1.0@dev"
+        "stack/callable-http-kernel": "~1.0@dev",
+        "phpunit/phpunit": "3.7.28"
```
Comment (cordoval) [inline composer.json]: you can just also do ~3.7

**Comment by cordoval** [comment]
wonder if we could ignore composer.lock in this case

File: composer.json (hunk 1 above)
Comment (igorw) [inline composer.json]: or `3.7.*` since PHPUnit tends to break stuff in minor versions.

**Comment by igorw** [comment]
@cordoval why?

**Comment by cordoval** [comment]
@igorw just because :baby: it helps reviewing the code better and take auto-generated things out of the way

**Comment by till** [comment]
@igorw Merge pahty? ;) (Also, I will probably be at Redwood Bar tonight. =)

**Pull request merged**